blog/modules/ROOT/pages/from-gitbook-to-antora.adoc

```diff
@@ -101,6 +101,12 @@ We also explicitly define the start page.
 By default, Antora looks for a file named `index.adoc` in the `ROOT` module.
 So if your start page is already named `index.adoc`, you can remove the `start_page` key.
 
+[NOTE]
+====
+The name of the component is determined from the `name` key in the component descriptor.
+In this migration guide we have named the directory the component descriptor is in `book` also, but this does not affect the name of the component.
```

Comment:
Sorry but I still don't get it 🤔 
"we have named the directory the component descriptor is in `book` also".

We have named the component's directory `book` ?
Component descriptor refers to the `antora.yml` file where we describe the component.

Reply:
Or we could say that the name of the directory does not matter when we reference the component. Only the name defined in the component descriptor (ie. the `antora.yml` file) matters.

Reply:
```suggestion
In this migration guide, the component root directory is named `book` but this does not affect the name of the component (defined in the component descriptor file).
```